cmake/thirdparty/SetupMPI.cmake

```diff
@@ -33,6 +33,29 @@ set(_mpi_link_flags )
 
 message(STATUS "Enable FindMPI:  ${ENABLE_FIND_MPI}")
 
+#
+# TODO: should this be BLT_ENABLE_FIND_MPI?
```

Comment:
> TODO: should this be `BLT_ENABLE_FIND_MPI`?

I agree that we should convert it to `BLT_ENABLE_FIND_MPI` and deprecate `ENABLE_FIND_MPI`, e.g. via a loud warning at first.

Perhaps we should do the same for many/most blt `ENABLE_` variables at the same time?

Reply:
It would be a good time to do it since we just did the 0.4.0 release.

Reply:
Added: https://github.com/LLNL/blt/issues/483